collect/collect.go

```diff
@@ -480,12 +480,12 @@ func (i *InMemCollector) processSpan(sp *types.Span) {
 	trace := i.cache.Get(sp.TraceID)
 	if trace == nil {
 		// if the trace has already been sent, just pass along the span
-		if sr, sentReason, found := i.sampleTraceCache.CheckSpan(sp); found {
+		if sr, keptReason, found := i.sampleTraceCache.CheckSpan(sp); found {
 			i.Metrics.Increment("trace_sent_cache_hit")
```

Comment:
maybe change the name of this metric? And are there other ones?

Reply:
I think this one is ok. This metric is for the `TraceSentCache`, not the `keptReasonCache`. Hopefully with this rename, things will be a bit more clear